blazor/treegrid/how-to/get-treegrid-properties.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Get the values of the Tree Grid properties associated with a record.| Syncfusion
+description: Learn here all about how to get the value of a record's tree grid properties in Syncfusion Blazor Tree Grid component and more.
+platform: Blazor
+control: Tree Grid
+documentation: ug
+---
+
+# Get row model details of the record in Blazor Tree Grid Component
+
+The values associated with row model details which includes the index, level, uniqueID, parent records, child records, expanded status, checkboxState, parentUniqueID of a record can be obtained by using the [GetRowModel](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeGrid.SfTreeGrid-1.html#Syncfusion_Blazor_TreeGrid_SfTreeGrid_1_GetRowModel) method of the Tree Grid component.
```

Comment:
Change it to a use case of checking the expanded status of a record

Reply:
use below sentence.

The Blazor Tree Grid Component provides a method called [GetRowModel](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeGrid.SfTreeGrid-1.html#Syncfusion_Blazor_TreeGrid_SfTreeGrid_1_GetRowModel) that can be used to obtain the values associated with row model details. These details include the index, level, unique ID, parent records, child records, expanded status, checkbox state, and parent unique ID of a record.